pandas/tests/groupby/test_timegrouper.py

```diff
@@ -385,7 +385,7 @@ def test_groupby_groups_datetimeindex(self):
         groups = grouped.groups
         assert isinstance(list(groups.keys())[0], datetime)
 
-        # GH 11442
+        # GH#11442
```

Comment:
not important, but was just wondering: why are you adding the '#' everywhere?

Reply:
Mostly for internal consistency (not a big deal, but for grepping purposes).  A little bit because I was curious how long it would take before someone asked about it.

Reply:
we have have the gh-xxxx style as well, slight prefernce for that